frontend/common/src/components/Pagination/Pagination.tsx

```diff
@@ -14,7 +14,7 @@ export interface PaginationProps {
   /** Total count of items from array. */
   totalCount: number;
   /** The minimum number of page buttons to be shown on each side of the current page button. */
-  siblingCount: number;
+  siblingCount?: number;
```

Comment:
If you are changing this to accept an undefined value then you must also change the [usePagination hook](https://github.com/GCTC-NTGC/gc-digital-talent/blob/feature/1683-prep-add-skills-to-experience/frontend/common/src/components/Pagination/usePagination.ts#L31) to match. The `Pagination.tests.tsx` tests are currently failing due to this.

Reply:
Oh, good catch. I really need to remember to run the tests!